pkg/services/sqlstore/stats_test.go

```diff
@@ -54,6 +54,13 @@ func TestStatsDataAccess(t *testing.T) {
 			err := GetAdminStats(&query)
 			assert.Nil(t, err)
 		})
+
+		t.Run("Get active user count stats should not result in error", func(t *testing.T) {
+			query := models.GetActiveUserCountStatsQuery{}
+			err := GetActiveUserCountStats(&query)
+			assert.Nil(t, err)
+			assert.Equal(t, query.Result.Count, int64(1))
```

Comment:
A bit backwards, but in `assert.Equal`, the reference value comes first:
```suggestion
			assert.Equal(t, int64(1), query.Result.Count)
```